pkg/gitreceive/storage/slug_builder_info.go

```diff
@@ -14,23 +10,18 @@ const (
 // slug builder.
 type SlugBuilderInfo struct {
 	pushKey string
-	pushURL string
 	tarKey  string
-	tarURL  string
 }
 
-// NewSlugBuilderInfo creates and populates a new SlugBuilderInfo based on the given data.
-func NewSlugBuilderInfo(s3Endpoint *Endpoint, bucket, appName, slugName string, gitSha *git.SHA) *SlugBuilderInfo {
+// NewSlugBuilderInfo creates and populates a new SlugBuilderInfo based on the given data
+func NewSlugBuilderInfo(slugName string) *SlugBuilderInfo {
 	tarKey := fmt.Sprintf("home/%s/tar", slugName)
-	// this is where the controller tells slugrunner to download the slug from, so we have to
-	// tell slugbuilder to upload it to here
-	pushKey := fmt.Sprintf("home/%s:git-%s/push", appName, gitSha.Short())
+	// this is where workflow tells slugrunner to download the slug from, so we have to tell slugbuilder to upload it to here
+	pushKey := fmt.Sprintf("home/%s/push", slugName)
```

Comment:
slugName is constructed using appname and gitSha. So removed since its just duplication of code.
https://github.com/deis/builder/blob/master/pkg/gitreceive/build.go#L56